hack/make-release.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# set vars 
+if [ -n "$1" ]; then
+    DOCKER_REPO=$1
+else
+    echo "Must provide a docker repo"
+    exit 
+fi
+
+if [ -n "$2" ]; then
+    TAG=$2
+else
+    echo "Must provide a version tag."
+    exit 
+fi
+
+if [ -n "$3" ]; then
+    BRANCH=$3
+else
+    echo "Must provide a git branch."
+    exit 
+fi
+
+# iterate over all services: build + push image, inject tag into k8s manifest. 
+for dir in ./src/*/    
```

Comment:
hmm use `find -type d` and even `-min-depth 1`, `-max-depth 1 for this, so that it doesn't include /src.